apps/certificate/certificate.py

```diff
@@ -0,0 +1,54 @@
+import os
+import io
+
+from bidi.algorithm import get_display
+from PIL import Image, ImageDraw, ImageFont
+
+from django.contrib.staticfiles import finders
+from django.utils.translation import gettext as _
+
+FONT_SIZE = 35
+FONT_COLOR  = (3, 8, 12)
+RIGHT_PADDING = 70
+
+def create_certificate_image(certificate):
+    # FIX: change to english strings and translate using gettext
+    TEMPLATE = [
+        _(f'שם מתנדב: {certificate.volunteer.first_name} {certificate.volunteer.last_name}'),
+        _(f'תעודת זהות: {certificate.volunteer.tz_number}'),
+        _(f'תוקף התעודה: {certificate.expiration_date}'),
+        _(f'מספר תעודה: {certificate.id}'),
```

Comment:
You shouldn't use user variables in the gettext function, as they shouldn't be translated. This should be named format placeholders that would be placed in using `format` functions.